client/src/redux/show-cert-saga.js

```diff
@@ -13,7 +12,8 @@ function* getShowCertSaga({ payload: { username, certSlug } }) {
       for (let i = 0; i < messages.length; i++) {
         yield put(createFlashMessage(messages[i]));
       }
-      yield call(navigate, '/');
+      // eslint-disable-next-line no-restricted-globals
+      history.go(-1);
```

Comment:
To be honest, I have no idea if this is a suitable option, but did not find another way without using `redirectQueryParams` which would get/look messy.

Ideas are welcome

Reply:
Honestly I'd leave this as is until we have a resolution for #45563

Reply:
@nhcarrigan To clarify: Are you suggesting blocking this PR until `45563` is resolved? Or, going ahead with this as is?

Reply:
I'm suggesting to leave this as `yield call(navigate, '/');` until we've implemented the FAQ page.